oauth-server/src/main/java/io/strimzi/kafka/oauth/server/NoopAuthorizer.java

```diff
@@ -0,0 +1,68 @@
+package io.strimzi.kafka.oauth.server;
+
+import kafka.network.RequestChannel;
+import kafka.security.auth.Acl;
+import kafka.security.auth.Authorizer;
+import kafka.security.auth.Operation;
+import kafka.security.auth.Resource;
+import org.apache.kafka.common.security.auth.KafkaPrincipal;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import scala.collection.immutable.Map;
+import scala.collection.immutable.Set;
+
+public class NoopAuthorizer implements Authorizer {
```

Comment:
Why do we need this class, we don't currently support authz?

Reply:
Removed from PR.